helix-core/src/main/java/org/apache/helix/BaseDataAccessor.java

```diff
@@ -261,5 +263,7 @@
   /**
    * Close the connection to the metadata store
    */
-  void close();
+  default void close() {
+
```

Comment:
Shall we print something such as "Default close method is invoked! No operation is executed." That can give user some hints.

Reply:
Updated.